src/xdp/program.c

```diff
@@ -691,8 +691,15 @@ XdpParseFrame(
             goto BufferTooSmall;
         }
         Cache->Ip4Hdr = (IPV4_HEADER *)&Va[Offset];
+        // Minimum value for a correct header is 5, as per RFC 791
+        if (Cache->Ip4Hdr->HeaderLength < 5) {
+            return;
```

Comment:
@mtfriesen Do you think this is worth a log?

Reply:
No, not really.